notebook/static/notebook/js/actions.js

```diff
@@ -211,21 +217,21 @@ define(function(require){
                 env.notebook.select(index);
             }
         },
-        'copy-selected-cell' : {
+        'copy-cell' : {
             icon: 'fa-copy',
             help_index : 'ef',
             handler : function (env) {
                 env.notebook.copy_cell();
             }
         },
-        'paste-cell-before' : {
+        'paste-cell-above' : {
```

Comment:
As mentioned above the differences between previous/next versus above/below are pretty subtle. However we are finding that we pick the ones that match the needed parts of speech. For example "paste cell next" is not a proper sentence where "past cell below" is. If you wanted to use "next" in a sentence you would have to say something like "paste cell into the next slot in the notebook", which is a much more cumbersome and verbose way of saying "paste cell below".